aviatrix-controller-initialize/variables.tf

```diff
@@ -69,6 +69,11 @@ variable controller_launch_wait_time {
   default     = 210
 }
 
+variable wait_time_for_instance {
+  description = "Wait time for controller instance to come up"
+  default     = 500
```

Comment:
Default 500 is bad and the controller would come up much earlier. Could we use 90, since this is customizable

Reply:
I tried 90, but initialization timed out. I used 500 just in case someone uses smaller instance. 
Let me try 200, 300 and 400.